script.js

```diff
@@ -1,4 +1,107 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+//game mode
+var diceRollMode = "dice rolling!";
+var diceOrderMode = "choose the order of dice";
+var beatThatWinner = "comparing the numbers";
+//default game mode
+var gameMode = diceRollMode;
```

Comment:
Nice breakdown of gameModes here! 

Would use scream case here to better separate this from the other variables 
e.g. 
```
DICEROLL_MODE = 'diceRolling'
gameMode = DICEROLL_MODE 
```